contribute/bugreporting.rst

```diff
@@ -56,8 +56,8 @@ Here we'll discuss the general steps to get logs, files that contain important d
 
     Log files may contain information you prefer to keep private. Please be sure to skim through them to ensure this is not the case before posting them. We can coordinate to get them through e-mail or a similarly less public medium.
 
-To get ready, download the `Logviewer app <https://open-store.io/app/logviewer.neothethird>`_ from the OpenStore. This app will help us find, view, and send the logs we need.
+To get ready, download the `Logviewer app <https://open-store.io/app/logviewer.ruditimmer>`_ from the OpenStore. This app will help us find, view, and send the logs we need.
 
-Now, open the Logviewer app. You'll see a list of all of the log files available in your user log folder, ``/home/phablet/.cache/upstart/``. This folder contains many, but not all, of the logs for software running on your device. You can tap on any log to view it, then tap the "send" icon in the top right corner to share it using an online service.
+Now, open the Logviewer app. You'll see a list of all of the services and applications log files running on your device. You can tap on any log to view it, then tap the "send" icon in the top right corner to share it using an online service.
```

Comment:
I think the sentence structure is a bit mangled. And maybe it can be said simpler. ... how about 

Now, open the Logviewer app. 
You'll see the list of applications running on your device. You can open any of them to view the logs of this application. And you can use the "send" icon in the top right corner to share a log using an online service.

Reply:
yes sounds good, will modify it